tests/test-program.py

```diff
@@ -194,38 +200,47 @@ def transform_messages(self, testcase, context=None):
 
     def test_basic(self):
         """ test basic message """
-        self.transform_messages('basic')
-        self.transform_messages_streaming('basic')
+        self.transform_messages({ 'testcase': 'basic'})
+        self.transform_messages_streaming({ 'testcase': 'basic'})
+
+    def test_basic_no_config(self):
+        """ test basic no config message """
+        schemas = {
+            'input': 'schemas/examples-messages.output.json',
+            'output': 'schemas/examples-messages-no-config.output.json',
+        }
+        self.transform_messages({ 'testcase': 'basic_no_config', 'schemas': schemas })
+        self.transform_messages_streaming({ 'testcase': 'basic_no_config'})
```

Comment:
Do we need to pass through the schemas here as well?

Reply:
We do, although clearly not for the test to fail.  Updated, and more tightly constrained the output schema for this particular case. 

064847b